### DISCUSSION TITLE
Which pins?

### DISCUSSION THREAD

**Discussion by Nickel1893** [tbnobody/OpenDTU#467] [Q&A] [answered]
![AD9DD6CD-0CF1-400A-8A85-41209D761D0B](https://user-images.githubusercontent.com/68815378/208910419-c40fe60c-98e6-445c-a0f9-552806c9c81e.jpeg)

I am a newbie :) having this board „ Espressif ESP32 WLAN Dev Kit Board Development Bluetooth Wifi v1 WROOM32 NodeMCU“. 

Can you tell me which Pins I need? Looks like it‘s a different board like in the schematic introduction. Thanks so far

**Comment by Solarteur** [comment, marked as answer]
I am a senior newbie :). In my opinion it should be possible to use the indicated pins and connect it to the nRF24L01+ module according to the schematic for the ESP32 38 pin. If experts disagree: please let us know
![ESP32_30_Pinout](https://user-images.githubusercontent.com/117977338/208959781-61ed12f0-7955-4106-bee8-8b4847897a59.png)

**Reply by smarthausleben** [reply]
I can confirm the proposed pin assignment because I’m using the same board (ESP32 DEV KIT V1) and exactly this pins and it works perfectly.

**Reply by Nickel1893** [reply]
Thank you both so much.
